integrations/agora-sdk-plugin.md

```diff
@@ -97,25 +97,21 @@ After activating the Symbl Conversation Intelligence Extension, click the **View
 
 This section walks you through the steps necessary to set up the Symbl Conversation Intelligence extension in your mobile application.
 
-1. Download the [Symbl Extension](https://cdn-agora.symbl.ai/agora-symblai-filter-debug.aar) (if you haven't already).
- 
-2. Add the `.aar` file as a dependency to your application. 
-![agora-creds](/img/agora-arr-files.png)
-
-3. Add the following information into your `build.gradle` module file:
+1. Add the following information into your `build.gradle` module file:
 
 ```js
 implementation fileTree(include: ['*.jar'], dir: 'libs')
 implementation 'com.squareup.okhttp3:okhttp:3.10.0'
 implementation 'org.java-websocket:Java-WebSocket:1.5.1'
+implementation 'ai.symbl:android.extension:0.0.1'
 ```
-4. Implement the interface io.agora.rtc2.IMediaExtensionObserver
+2. Implement the interface io.agora.rtc2.IMediaExtensionObserver
  
 ```js
 public class MainActivity extends AppCompatActivity implements io.agora.rtc2.IMediaExtensionObserver {
 ```
 
-5. Add the following method to set all the necessary information to initialize the Symbl configuration. You can find description for the parameters used in the table below:
+3. Add the following method to set all the necessary information to initialize the Symbl configuration. You can find description for the parameters used in the table below:
```

Comment:
Hey @visharma-symbl I think maybe a line or so clarifying what MainActivity file is might be helpful, just something about it's role, where it would most probably be placed in the general structure of an android application. 
Similarly for the interface we are implementing (io.agora.rtc2.IMediaExtensionObserver). If there are any existing sections in agora docs for this, we could simply link them in. 
Would all this make sense, or would it be redundant? CC: @mjabali

Reply:
Hi @janhavi-ai,
I guess these will be self-explanatory to the end-users/developers.

- **MainActivity:** MainActivity in android terminology is the first screen/activity that gets executed.
It is just like the `main()` method in other programming terms. 

- **IMediaExtensionObserver:** This is very specific to Agora SDK. Also, I was finding the document related to this on Agora Platform but couldn't find one as they have updated their documentation.  _(Earlier, there was documentation on Agora regrading this)_

Let me know in case you have any doubts.